airflow/models/dag.py

```diff
@@ -2819,8 +2819,6 @@ class DagModel(Base):
     # Time when the DAG last received a refresh signal
     # (e.g. the DAG's "refresh" button was clicked in the web UI)
     last_expired = Column(UtcDateTime)
-    # Whether (one  of) the scheduler is scheduling this DAG at the moment
-    scheduler_lock = Column(Boolean)
```

Comment:
we would also need a SQLAlchemy migration script generated using Alembic & kept in https://github.com/apache/airflow/tree/main/airflow/migrations/versions to drop this column, no?

Reference guide to generate the migration script: https://github.com/apache/airflow/blob/main/contributing-docs/13_metadata_database_updates.rst

cc: @ephraimbuddy

Reply:
Added the migration script.